wiki/Play_style/en.md

```diff
@@ -95,13 +95,13 @@ There are two main styles of setting the keys placement in the keyboard: KDDK (T
 
 ### Taiko controller
 
-A taiko controller is the default play style and controller for the [Taiko no Tatsujin](https://wikipedia.org/wiki/Taiko_no_Tatsujin) series([Taiko Drum Master](https://wikipedia.org/wiki/Taiko:_Drum_Master) for US).
+A taiko controller is the default play style and controller for the [Taiko no Tatsujin](https://wikipedia.org/wiki/Taiko_no_Tatsujin) series ([Taiko Drum Master](https://wikipedia.org/wiki/Taiko:_Drum_Master) for US).
 
 *Note: Some skilled players who play using a Taiko controller may create theirs from scratch while others purchase theirs via retail or online stores.*
 
 Players who use a Taiko controller carry a Taiko/drum stick in each hand and hit the centre (Red) or rim of the drum (Blue) to register the note based on where it is hit.
 
-*Notice: Most Taiko controllers connect to the computer by using a Bluetooth connector should there be no USB connector. Though this may increase latency*
+*Notice: Most Taiko controllers connect to the computer by using a Bluetooth connector should there be no USB connector. Though this may increase latency.*
```

Comment:
```suggestion
*Notice: Most Taiko controllers connect to the computer by using a Bluetooth connector should there be no USB connector, though this may increase latency.*
```